specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -14073,7 +16496,7 @@
         },
         "allocatedMemoryInMB": {
           "format": "double",
-          "description": "The allocated memory in MB.",
+          "description": "The allocated mememory in MB.",
```

Comment:
memory

Reply:
done

---
In reply to: [588104998](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r588104998) [](ancestors = 588104998)